apps/openmw/mwgui/widgets.cpp

```diff
@@ -532,12 +532,11 @@ namespace MWGui
             assignWidget(mBarTextWidget, "BarText");
         }
 
-        MWScrollBar::MWScrollBar()
-            : mEnableRepeat(true)
-            , mRepeatTriggerTime(0.5f)
-            , mRepeatStepTime(0.1f)
-            , mIsIncreasing(true)
+        MWScrollBar::MWScrollBar() : mIsIncreasing(true)
         {
+            mEnableRepeat = true;
+            mRepeatTriggerTime = 0.5f;
+            mRepeatStepTime = 0.1f;
```

Comment:
This change looks pointless.

Reply:
This variables were in parent MyGUI class and duplicated in MWScrollBar class. I removed them, and since I can't init them with member init-list anymore, I did it in ctor. Now I'm reverting this because of back-compatibility, as @scrawl mentioned.